### PR TITLE
Localize extension

### PR DESCRIPTION
**Pull request by Faizan2304**
Issue: Extension folder doesn't have resource dlls 
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/510774 
Fix: Copy resource dlls in extension folder. Update TestPlatform.csproj to include this in vsix and update sign proj.

### CODE REVIEW

**Review by harshjain2 — APPROVED** [review]
Can we add Acceptance Test to ensure that Localization is working?

**Comment by harshjain2** [comment]
Please share the validations that you have performed.

File: scripts/build.ps1 [hunk 1]
```diff
@@ -354,6 +378,32 @@ function Publish-PackageInternal($packagename, $framework, $output)
     & $dotnetExe publish $packagename --configuration $TPB_Configuration --framework $framework --output $output -v:minimal -p:Version=$TPB_Version -p:CIBuild=$TPB_CIBuild -p:LocalizedBuild=$TPB_LocalizedBuild
 }
 
+function Copy-Loc-Files($sourceDir, $destinationDir, $dllName)
+{
+    $language = @("cs", "de", "es", "fr", "it", "ja", "ko", "pl", "pt-BR", "ru", "tr", "zh-Hans", "zh-Hant")
```
Comment (singhsarab) [inline scripts/build.ps1]: DRY

**Review by singhsarab — APPROVED** [review]
With suggestions.

File: scripts/build.ps1 (hunk 1 above)
Comment (Faizan2304) [reply scripts/build.ps1]: Resolved

**Comment by Faizan2304** [comment]
@dotnet-bot  test Windows_NT / Debug Build please

**Pull request merged**